lib/spark.js

```diff
@@ -16,6 +17,7 @@ export default class Spark {
 
   constructor ({ fetch = globalThis.fetch } = {}) {
     this.#fetch = fetch
+    this.lookupMinerPeerId = lookupMinerPeerId
```

Comment:
I am fine either way.

The downside of your proposal is that I have to rename the imported implementation of `getMinerPeerId`, otherwise I get the following error:


```
error: ReferenceError: Cannot access 'getMinerPeerId' before initialization
    getMinerPeerId = getMinerPeerId,
                     ^
    at new Spark (file:///Users/bajtos/src/filstation/spark/lib/spark.js:21:22)
```

Reply:
See 7378e8d

Reply:
See [7378e8d](https://github.com/filecoin-station/spark/pull/55/commits/7378e8d775cce52754615260a0a60f89d921c65b)

Reply:
Yeah that's unfortunate, but, to me, conceptionally it's cleaner to rename a variable than to use two methods for mocking